GovUk.Frontend.AspNetCore.Extensions/Views/Shared/GOVUK/Pagination.cshtml

```diff
@@ -0,0 +1,90 @@
+﻿@using System.Globalization;
+@using System.Web;
+@addTagHelper *, GovUk.Frontend.AspNetCore
+@model GovUk.Frontend.AspNetCore.Extensions.Models.PaginationModel
+@if (Model != null && Model.PageSize.HasValue && Model.TotalItems > Model.PageSize)
```

Comment:
Is it better to use `Model is not null` rather than `Model != null` as the `is` operator doesn't run any custom equality operators. We could create our own equality operator such that `var a = new Model(); a == null` is `true`. Not that any sane person would do that but it is possible.

I can see that we are targeting net6.0 so backwards compatibility would not be a problem.

Reply:
That's an interesting C#9.0 feature I didn't know, but in this case as you say it's not going to be important. We're in control of the model class it's checking and we know it doesn't have an overridden equality operator.